.azure-pipelines/linux-conda-CI.yml

```diff
@@ -19,10 +19,11 @@ jobs:
         numpy.version: '>=1.18.1'
         onnx.version: '==1.7.0'
         onnx.target_opset: ''
-        onnxrt.version: '-i https://test.pypi.org/simple/ ort-nightly'
+        onnxrt.version: 'onnxruntime==1.3.0'  # -i https://test.pypi.org/simple/ ort-nightly'
```

Comment:
Could you please clean-up commented out code?

Reply:
I let them on purpose to have an easy to way to test against nightly build as I never remember this syntax.

Reply:
Not sure if that's a good reason to have commented code. You can always look at the history and see the previous version.

Reply:
I'll remove it but my point is people look for something in history if they think can find it there. I doubt this would be the case for this.